src/Constants.h

```diff
@@ -95,6 +96,16 @@ constexpr const char* ARDUPILOT_PROTOCOL_NAME = "/ardupilot";
 constexpr std::chrono::milliseconds JOINT_POWER_REPEAT_PERIOD(333);
 constexpr std::chrono::milliseconds ARM_IK_UPDATE_PERIOD(50);
 
+constexpr std::array<robot::types::motorid_t, 4> WHEEL_IDS = {
+	robot::types::motorid_t::frontLeftWheel,
+	robot::types::motorid_t::frontRightWheel,
+	robot::types::motorid_t::rearLeftWheel,
+	robot::types::motorid_t::rearRightWheel,
```

Comment:
Just a note, these motors are the actual wheels, we should add `motorid_t` values for the steer motors. I think Victor should handle that in his PR, though, so wait for that and you can merge it in